src/Components/Alert/Components/Filters/Price.tsx

```diff
@@ -18,11 +32,70 @@ export const Price: React.FC = () => {
 
   return (
     <>
-      <Text variant="sm-display">Price Range</Text>
+      <Text variant="sm-display" mb={2}>
+        Price Range
+      </Text>
+
       <PriceRange
+        bars={bars}
         priceRange={state.criteria.priceRange || DEFAULT_PRICE_RANGE}
         onDebouncedUpdate={handlePriceRangeUpdate}
       />
     </>
   )
 }
+
+export const PriceFragmentContainer = createFragmentContainer(Price, {
+  aggregations: graphql`
+    fragment Price_aggregations on FilterArtworksConnection {
+      aggregations {
+        slice
+        counts {
+          name
+          value
+          count
+        }
+      }
+    }
+  `,
+})
+
+export const PriceQueryRenderer = () => {
+  const { state } = useAlertContext()
+
+  const artistID = state.criteria.artistIDs?.[0]
+
+  if (!artistID) return <Price />
+
+  return (
+    <SystemQueryRenderer<PriceAggregationsQuery>
+      lazyLoad
+      placeholder={<Price />}
+      variables={{ artistID }}
+      query={graphql`
+        query PriceAggregationsQuery($artistID: String!) {
+          artist(id: $artistID) {
```

Comment:
**suggestion**: Instead of using `filterArtworksConnection` within the Artist type, which also means we need to fetch the artist separately, are we able to use the root-level artworksConnection query field instead? That would also allow us to easily thread multiple artistIDs into the query for the infrequent case when an initial alert criteria includes multiple artists, as is this case on some artwork pages:

```graphql
{
  artworksConnection(first: 0, artistIDs: $artistIDs, aggregations: [SIMPLE_PRICE_HISTOGRAM]) {
    ...Price_aggregations
  }
}
```

Reply:
I just tried to simulate this with multiple artist IDs and I get the following error from Gravity:

```json
{
  "error": "Price histogram aggregation requires an artist or artist series filter"
}
```

but I feel like we could easily adjust the condition [here](https://github.com/artsy/gravity/blob/15330a2d859281d2c71f9a22607e416657e5aa0e/app/api/v1/filter_endpoint.rb#L93) to also support queries with `artist_ids` supplied. In the meantime (until we discuss and action that Gravity change) we can pluck the first artist ID from the alert context but still use the root-level query field.

Reply:
If you're ok with it, I'll add a TODO comment, merge this PR and then do the Gravity change.

Reply:
Is it possible to at least switch to the root-level query field but pass the first artist ID, instead of multiple, before merging? Then yeah, I think we can follow-up on that potential Gravity change to support a histogram backed by multiple artist ID arguments.

Reply:
Gotcha! I've changed the query to use the root artworks connection 👍